src/racingCar/CarRacing.js

```diff
@@ -0,0 +1,59 @@
+import { Console } from "@woowacourse/mission-utils";
+import Car from "./Car.js";
+import HandleIo from "./HandleIo.js";
+import ErrorChecker from "../checker/ErrorChecker.js";
+
+class CarRacing {
+    async run() {
+
+        const handleinput = new HandleIo();
+        const { nameInput, tryCount } = await handleinput.getInput();
+
+        this.startRacing(nameInput, tryCount);
+    };
+
+    startRacing(nameInput, tryCount) {
+
+        const cars = this.createCar(nameInput);
+
+        if (!cars) return;
+
+        ErrorChecker.checkTryCount(tryCount);
+
+        for (let i = 0; i < tryCount; i++) {
+            cars.forEach(car => car.move());
+            Console.print("\n");
+        };
+
+        this.checkWinner(cars);
+    };
+
+    createCar(input) {
+        const cars = input.split(',').map((name) => {
+            name = name.trim();
+            ErrorChecker.checkName(name);
+            return new Car(name);
```

Comment:
Car클래스를 생성하면서 Car클래스의 컨스트럭터나 create라는 형태의 메소드를 통해서 객체를 생성하는 팩토리 패턴을 이용해 이름에 대해서 검증했으면 아래처럼 바꿔쓸 수 있지 않을까 싶어요!

```suggestion
            return new Car(name.trim()) // or new Car.create(name.trim()); 
```